initializers/sequelize.js

```diff
@@ -59,9 +73,14 @@ module.exports = {
 
       autoMigrate: function(next) {
         if(api.config.sequelize.autoMigrate == null || api.config.sequelize.autoMigrate) {
-            api.sequelize.migrate({method: 'up'}, function () {
-              next();
-            });
+            migrateSequelizeMeta(api, umzug)
+                .then(function() {
+                    return umzug.up().then(function() {
+                        return;
```

Comment:
You're right, it isn't necessary - was me being derpy and not understanding Promises properly ;p